pydantic_resolve/resolver.py

```diff
@@ -149,7 +151,10 @@ def _execute_resolver_method(self, method):
                         # if extra transform provides
                         loader = Loader()
 
-                        filter_config = self.loader_filters.get(Loader, {})
+                        filter_config = {
```

Comment:
additional check required.

if same field is set by two sources, it's better to raise Exception  (eg: LoaderFieldConlictError)

```python
Resolver(loader_global_filters={"lang": "en"}, loader_filters={LoaderA: {"lang": "cn"}})   # raise error
```


```python
Resolver(loader_global_filters={"lang": "en"}, loader_filters={LoaderA: {"country": "cn"}})   # no error
```

it would be easier to be aware of the final form of filters.